docs/tools-reference/custom-kernels-distros/custom-distro-kvm.md

```diff
@@ -0,0 +1,287 @@
+
+Using Xen? [See this guide.](https://www.linode.com/docs/tools-reference/custom-kernels-distros/running-a-custom-linux-distro-on-a-linode-vps)
+
+See the [Addendum](#add) at the end of the guide for more info and QAs.
+
+##Creating the Virtual Machine Instance
+- Setting the default Linode instance hypervisor:
+	- Log in to the Linode Manager.
+	- Click on the **Account** tab.
+	- Under the **Hypervisor Preference** header, ensure **KVM** is selected.
+	- Click the **Save** button. 
+
+	[![KVM Hypervisor Preference](/docs/assets/snap001.png)](/docs/assets/snap001.png)
+
+- Create the new Linode instance:
+	- Click on the **Linodes** tab.
+	- Click on the **Add a Linode** button
+	- Select the preferred plan and instance location.
+	- Click the **Add this Linode!** button.
+
+- Disable Lassie:
+	- Click on the name of the newly created instance.
+	- Click on the **Settings** tab.
+	- Under Shutdown Watchdog, set Lassie is currently to **disabled**.
+	- Click on the **Save Changes** button
+
+	[![Lassie Disabled](/docs/assets/snapresize0014.png)](/docs/assets/snapresize0014.png)
+
+
+##Creating the System Disks
+One temporary disk will be needed for the installation media, and as many disks as needed to achieve your preferred system layout. In this guide, we will keep it simple and use only one disk for the entire system.
+
+- Click on the **Dashboard** sub-tab.
+- Under the **Disks** header, Click on **Create a new Disk**.
+- Under the **Edit Disk** header, enter **media** as the name.
+- Select **ext4** as the type.
+- Enter **512MB** as the size.
+- Click the **Save Changes** button.
+
+	[![Media Disk](/docs/assets/snap0004.png)](/docs/assets/snap0004.png)
+- Click on **Create a new Disk** again
+- Enter **system** as the name.
+- Select **ext4** as the type.
+- Enter **8192MB** as the size.
+- Click on the **Save Changes** button.
+
+	[![System Disk](/docs/assets/snap0005.png)](/docs/assets/snap0005.png)
+
+##Loading the Installation Media
+- Click on the **Rescue** tab.
+- Make a note of path to disk mappings. In this guide **/dev/sda** is the **media** disk, **/dev/sdb** is the **system** disk.
+- Click on the **Reboot Into Rescue Mode** button.
+
+	[![Rescue Mode](/docs/assets/snap0006.png)](/docs/assets/snap0006.png)
+- Click on the **Remote Access** tab.
+- Under the **Console Access** header, click on the **Launch Lish Ajax Console** link.
+
+	[![Launch Lish](/docs/assets/snap0007.png)](/docs/assets/snap0007.png)
+- Enter the following commands into the Lish Web Console:
+	1. mkdir /mnt/system	
+	2. mount /dev/sdb /mnt/system
+	3. cd /mnt/system
+	4. wget http://less.cogeco.net/CentOS/6.6/isos/x86_64/CentOS-6.6-x86_64-netinstall.iso
+		> Alternate mirrors: http://www.centos.org/download/mirrors/
+	5. dd if= CentOS-6.6-x86_64-netinstall.iso of=/dev/sda
```

Comment:
Should be `dd if=CentOS-6.6-x86_64-netinstall.iso of=/dev/sda`

There's a misplaced space character before the ISO filename.